docs/design/ovs-pipeline.md

```diff
@@ -307,6 +334,28 @@ The table-miss flow entry (flow 3) will drop all other packets. This flow should
 never be used because only ARP traffic should go to this table, and
 ARP traffic will either match flow 1 or flow 2.
 
+### ServiceHairpinTable (23)
+
+If a Pod accesses a Service and the Pod is selected as the Endpoint of the Service,
```

Comment:
When a backend Pod of a Service accesses the Service, and the Pod itself is selected as the destination, then we have the hairpin case, in which the source IP should be...